WebRTC-server/main.py

```
@@ -0,0 +1,175 @@
+active_camera = None
+
+# Socket.IO events
+@sio.event
+async def connect(sid, environ):
+    logger.info(f"Client connected: {sid}")
+
+@sio.event
+async def disconnect(sid):
+    logger.info(f"Client disconnected: {sid}")
+    if sid in clients:
+        client_type = clients[sid]['type']
```

Ruff (0.12.2)

Local variable `client_type` is assigned to but never used

Remove assignment to unused variable `client_type`

(F841)